jena-base/src/main/java/org/apache/jena/atlas/io/BlockUTF8.java

```diff
@@ -28,7 +28,7 @@
  * This code is just the UTF-8 encoding rules - it does not check for legality
  * of the Unicode data.  The standard codecs do, so do not round-trip with binary
  * compatibility. (Example: a single element of a surrogate pair will
- * be encoded/decoded without lost.)
+ * be encoded/decoded without lose.)
```

Comment:
Or loss? (not a native, I might be wrong :+1)

Reply:
!!